app/Services/Post.php

```diff
@@ -153,6 +154,17 @@ public static function getPostFromUrl( $postUrl ) {
 		// Create the ACF Widgets from DOM content
 		$widgets = Widget::getWidgetsFromDom( $postDom );
 
+		// if there is a video header, convert to embed widget
+		if ($hero == "hero-without-media") {
+			$headerEmbed = Video::getVideoFromHeader($postObject->provider, $postObject->videoId);
+			array_unshift($widgets, $headerEmbed);
+			/**
+			* @var $_logger \Croissant\DI\Dependency\CatfishLogger
+			*/
+			$_logger = \Croissant\App::get( \Croissant\DI\Interfaces\CatfishLogger::class );
+			$_logger->notice("provider: " . $postObject->provider . ", video id: " . $postObject->videoId . ", url: " . $postUrl);
+		}
```

Comment:
ah does it work differently in production? Cause it was logging fine on my local?